modules/nodes-safe-sysctls-list.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+:_content-type: REFERENCE
+[id="safe_and_unsafe_sysctls_{context}"]
+= Safe and unsafe sysctls
+
+Sysctls are grouped into _safe_ and _unsafe_ sysctls.
+
+For system-wide sysctls to be considered safe, they must be namespaced. A namespaced sysctl ensures there is isolation between namespaces and therefore pods. If you set a sysctl for one pod it must not add any of the following:
+
+- Influence any other pod on the node
+- Harm the node health
+- Gain CPU or memory resources outside of the resource limits of a pod
+
+[NOTE]
+====
+Being namespaced alone is not sufficient for the sysctl to be considered safe.
+====
+Any sysctl that is not added to the allowed list on {product-title} is considered unsafe for {product-title}.
+
+Unsafe sysctls are not allowed by default. For system-wide sysctls the cluster administrator must manually enable them on a per-node basis. Pods with disabled unsafe sysctls are scheduled but do not launch.
+
+[NOTE]
+====
+You cannot manually enable interface-specific unsafe sysctls.
+====
+
+{product-title} adds the following system-wide and interface-specific safe sysctls to an allowed safe list:
+
+.System-wide safe sysctls
+[cols="30%,70%",options="header"]
+|===
+| sysctl | Description
+
+| `kernel.shm_rmid_forced`
+a|When set to `1`, all shared memory objects in current IPC namespace are automatically forced to use IPC_RMID. For more information, see link:https://docs.kernel.org/admin-guide/sysctl/kernel.html?highlight=shm_rmid_forced#shm-rmid-forced[shm_rmid_forced].
+
+| `net.ipv4.ip_local_port_range`
+a| Defines the local port range that is used by TCP and UDP to choose the local port. The first number is the first port number, and the second number is the last local port number. If possible, it is better if these numbers have different parity (one even and one odd value). They must be greater than or equal to `ip_unprivileged_port_start`. The default values are `32768` and `60999` respectively. For more information, see link:https://docs.kernel.org/networking/ip-sysctl.html highlight=ip_local_port_range#ip-variables[ip_local_port_range].
```

Comment:
This link isn't rendering properly, since there is a space in it

![image](https://user-images.githubusercontent.com/7443887/179522395-a1b5f0d3-aadf-4c2a-a074-04e25503e54f.png)